### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MrAE**

### CODE REVIEW

**Comment by MrAE** [comment]
NB, the urerf test failed 0.2 > 0.9

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ![Downloads shield](https://img.shields.io/pypi/dm/rerf.svg)
 
 
-RerF (aka Randomer Forest or Random Projection Forests) is an algorithm developed by [Tomita et al. (2016)](https://arxiv.org/abs/1506.03410) which is similar to Random Forest-Random Combination (Forest-RC) developed by [Breiman (2001)](https://doi.org/10.1023/A:1010933404324).  
+SPORF (aka RerF, Randomer Forest or Random Projection Forests) is an algorithm developed by [Tomita et al. (2016)](https://arxiv.org/abs/1506.03410) which is similar to Random Forest-Random Combination (Forest-RC) developed by [Breiman (2001)](https://doi.org/10.1023/A:1010933404324).
```
Comment (falkben) [inline README.md]: we should say "sparse projection oblique randomer forests" here as well...

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# [RerF](https://neurodata.io/tools)
+# [SPORF/RerF](https://neurodata.io/tools)
```
Comment (falkben) [inline README.md]: The link here should go to neurodata.io/sporf i think

**Review by falkben — CHANGES_REQUESTED** [review]
see comments

**Review by falkben — APPROVED** [review]

**Pull request merged**